app/ui/src/app/customizations/api-connector/api-connector.models.ts

```diff
@@ -1,14 +1,11 @@
-import { BaseEntity } from '@syndesis/ui/model';
 import { StringMap, BaseReducerModel } from '@syndesis/ui/platform';
 
-export interface ApiConnector extends BaseEntity {
+export interface ApiConnector  {
```

Comment:
Actually It is okay to extend ApiConnector from BaseEntity, since it is supposed to feature the two properties enforced by BaseEntity’s interface scheme (id and kind).

Reply:
Makes sense!